tests/analysis/damages/test_damages.py

```diff
@@ -259,13 +258,10 @@ def test_event_based_damage_calculation_manual_stylized(self):
         ]
 
         # LOAD DAMAGE FUNCTIONS
-        manual_damage_functions = ManualDamageFunctions()
-        manual_damage_functions.find_damage_functions(
-            folder=damages_test_data / "test_damage_functions"
+        manual_damage_functions = ManualDamageFunctionsReader().read(
```

Comment:
good that you are updating this, I'd suggest to add an `isinstance` over the result of this `read` operation.

Reply:
Not sure if I agree.
In fact this is part of the step "define test data". Other tests cover the result of the read method in `test_manual_damage_functions_reader.py`.
Added anyway.